test/ci_workflow_gen.go

```diff
@@ -26,11 +26,33 @@ import (
 	"text/template"
 )
 
+type mysqlVersion string
+
+const (
+	mysql57    mysqlVersion = "mysql57"
+	mysql80    mysqlVersion = "mysql80"
+	mariadb102 mysqlVersion = "mariadb102"
+	mariadb103 mysqlVersion = "mariadb103"
+
+	defaultMySQLVersion mysqlVersion = mysql57
+)
+
+type mysqlVersions []mysqlVersion
+
+var (
+	defaultMySQLVersions   = []mysqlVersion{defaultMySQLVersion}
+	mysql80OnlyVersions    = []mysqlVersion{mysql80}
+	allOracleMySQLVersions = []mysqlVersion{mysql57, mysql80}
```

Comment:
Pedantic, but I would not say OracleMySQL here:
1. There's just MySQL (Oracle is the company, MySQL is the product)
2. We actually use Percona Server for MySQL in most cases

Reply:
Fixed!